src/components/SuggestedJourneys/SuggestedJourneys.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { stationsById } from "stations";
+import { JourneyParams } from "types";
+
+import styles from "./SuggestedJourneys.module.scss";
+
+interface journey {
+    // origin: string;
+    // destination: string;
+    // originStationId: JourneyParams["fromStationId"];
+    // destStationId: JourneyParams["toStationId"];
+    // day: JourneyParams["day"];
+    // time: JourneyParams["time"];
+    params: JourneyParams;
```

Comment:
I think you can use the `JourneyParams` interface directly in place of the `journey` one you've written here. Rather than specifying the station names alongside a `JourneyParams`, you can extract them from `fromStationId` and `toStationId` with:

```ts
const fromStationName = stationsById[fromStationId].name
```
using the `stationsById` object from `stations.ts`